packages/web-app-admin-settings/src/mixins/groups/delete.ts

```diff
@@ -0,0 +1,96 @@
+import { mapActions } from 'vuex'
+import { clientService, configurationManager, eventBus } from 'web-pkg'
+
+export default {
+  computed: {
+    $_delete_items() {
+      return [
+        {
+          name: 'delete',
+          icon: 'close-circle',
```

Comment:
What's the reason for changing the icon? Was `delete-bin` before and I found that to be more intuitive.